hdk-rust/src/api.rs

```diff
@@ -326,31 +316,32 @@ pub fn link_entries<S: Into<String>>(
     let mut mem_stack = unsafe { G_MEM_STACK.unwrap() };
 
     // Put args in struct and serialize into memory
-    let input = LinkEntriesArgs {
-        base: base.clone(),
-        target: target.clone(),
-        tag: tag.into(),
-    };
-
-    let allocation_of_input = store_as_json(&mut mem_stack, input)
-        .map_err(|err_code| ZomeApiError::Internal(err_code.to_string()))?;
+    let allocation_of_input = store_as_json(
+        &mut mem_stack,
+        LinkEntriesArgs {
+            base: base.clone(),
+            target: target.clone(),
+            tag: tag.into(),
+        },
+    )?;
 
     let encoded_allocation_of_result: u32 =
         unsafe { hc_link_entries(allocation_of_input.encode() as u32) };
 
     // Deserialize complex result stored in memory and check for ERROR in encoding
-    let result: LinkEntriesResult = load_json(encoded_allocation_of_result as u32)
-        .map_err(|err_str| ZomeApiError::Internal(err_str))?;
+    let result: ZomeApiInternalResult = load_json(encoded_allocation_of_result as u32)?;
+
+    println!("link_entries result {:?}", result);
```

Comment:
Hm, I'd be surprised if this would work here. Inside the zome you need to use `hdk::debug()` to get an output..